UniConnectApp/app/src/main/java/com/example/uniconnect/MainActivity.kt

```diff
@@ -57,28 +59,76 @@ class MainActivity : ComponentActivity() {
+                style = MaterialTheme.typography.h6,
+                modifier = Modifier.padding(start = 20.dp)
             )
-            OutlinedTextField(
-                value = description,
-                onValueChange = {description =it},
-                label = { Text(stringResource(R.string.description))}
-            )
-            Button(
-                onClick = {
-                    var post = Post(title = title, description = description)
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(130.dp)
+                    .padding(horizontal = 30.dp),
+                horizontalArrangement = Arrangement.Center,
+            ) {
+                OutlinedTextField(
+                    value = title,
+                    onValueChange = {title = it},
+                    label = { Text(stringResource(R.string.title))},
+                    shape = RoundedCornerShape(20.dp),
+                    modifier = Modifier
+
+                        .fillMaxWidth()
+                )
+            }
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .height(500.dp)
+                    .padding(horizontal = 30.dp),
+                horizontalArrangement = Arrangement.Center,
+            ) {
+                OutlinedTextField(
+                    value = description,
+                    onValueChange = {description = it},
+                    label = { Text(stringResource(R.string.description))},
+                    shape = RoundedCornerShape(20.dp),
+                    modifier = Modifier
+                        .fillMaxSize()
+                )
+            }
+            Row(
+                horizontalArrangement = Arrangement.SpaceBetween,
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 30.dp, horizontal = 30.dp)
+            ){
+                Button(
+                    onClick = {
+                        var post = Post(title = title, description = description)
 
-                    viewModel.savePost(post)
-                    //Toast.makeText(context, ", $title, $description", Toast.LENGTH_LONG).show()
+                        viewModel.savePost(post)
+                        Toast.makeText(context, ", $title, $description", Toast.LENGTH_LONG).show()
+                    },
+                    shape = RoundedCornerShape(20.dp)
+                ){
+                    Text(text = "Post", Modifier.padding(start = 10.dp))
                 }
-            ){Text(text = "Post")}
-            Button (onClick = { signOn() })
-                { Text(text = "Logon") }
+
+                Button (onClick = {
+                    signOn()
+                },
+                    shape = RoundedCornerShape(20.dp)
+                ) {
+                    Text(text = "Logon", Modifier.padding(start = 10.dp))
+                }
```

Comment:
Buttons are fundamentally the same. I just rounded them to better reflect the story board.